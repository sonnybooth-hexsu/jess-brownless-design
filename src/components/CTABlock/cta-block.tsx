import React from "react"
import { Link } from "gatsby"
import buttons from "../../styles/buttons.module.css"
import containerClasses from "../../styles/container.module.css"

export const CTABlock = () => {
  return (
    <div className="text-white bg-Brand-green">
      <div className={`${containerClasses.container} py-24 sm:py-20 lg:py-40`}>
        <div className="grid grid-cols-12 gap-4">
          <div className="col-span-12 text-center lg:col-span-8 lg:col-start-3 xl:col-span-6 xl:col-start-4">
            <h2 className="text-white headingResponsive02">
              Book a consultation
            </h2>
            <p className="col-span-12 mt-6 mb-8 text-white lg:col-span-6 lg:col-start-4">
              If you would like to make an appointment or have an enquiry,
              please contact Miss Hawkes.
            </p>
            <Link
              className={`inline-block mt-0 lg:self-start ${buttons.btnOutlineBright}`}
              to="/contact"
            >
              <div className="flex items-center justify-between">
                Contact Miss Hawkes
              </div>
            </Link>
          </div>
        </div>
      </div>
    </div>
  )
}
