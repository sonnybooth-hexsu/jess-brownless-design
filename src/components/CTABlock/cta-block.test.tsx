/* eslint no-undef: 0 */

import React from "react"
import { render } from "@testing-library/react"
import { CTABlock } from "./cta-block"

describe("<CTABlock />", () => {
  it("should match snapshot", () => {
    const { container } = render(<CTABlock />)

    expect(container).toMatchSnapshot()
  })
})
