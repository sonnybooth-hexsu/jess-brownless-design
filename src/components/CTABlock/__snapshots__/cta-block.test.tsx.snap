// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CTABlock /> should match snapshot 1`] = `
<div>
  <div
    class="text-white bg-Brand-green"
  >
    <div
      class="container py-24 sm:py-20 lg:py-40"
    >
      <div
        class="grid grid-cols-12 gap-4"
      >
        <div
          class="col-span-12 text-center lg:col-span-8 lg:col-start-3 xl:col-span-6 xl:col-start-4"
        >
          <h2
            class="text-white headingResponsive02"
          >
            Book a consultation
          </h2>
          <p
            class="col-span-12 mt-6 mb-8 text-white lg:col-span-6 lg:col-start-4"
          >
            If you would like to make an appointment or have an enquiry, please contact Miss Hawkes.
          </p>
          <a
            class="inline-block mt-0 lg:self-start btnOutlineBright"
            href="/contact"
          >
            <div
              class="flex items-center justify-between"
            >
              Contact Miss Hawkes
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
